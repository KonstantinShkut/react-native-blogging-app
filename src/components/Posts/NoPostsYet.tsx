import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

export const NoPostsYet = () => {
  return (
    <View style={styles.container}>
      <Text>You don't have any posts yet.</Text>
      <Text> </Text>
      <Text>
        Start by tapping the Plus button at the bottom to create your first
        post.
      </Text>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    alignItems: 'center',
    padding: 40,
  },
});
