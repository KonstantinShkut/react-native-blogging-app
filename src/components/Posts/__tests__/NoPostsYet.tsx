import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import { NoPostsYet } from '../NoPostsYet';

describe('NoPostsYet', () => {
  it('renders correctly', () => {
    const component = renderer.create(<NoPostsYet />);
    expect(component).toMatchSnapshot();
  });
});
