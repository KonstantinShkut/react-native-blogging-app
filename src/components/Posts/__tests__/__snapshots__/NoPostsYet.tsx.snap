// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoPostsYet renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "padding": 40,
    }
  }
>
  <Text>
    You don't have any posts yet.
  </Text>
  <Text>
     
  </Text>
  <Text>
    Start by tapping the Plus button at the bottom to create your first post.
  </Text>
</View>
`;
